src/components/MyPage.js

```diff
@@ -84,8 +84,9 @@ class MyPage extends Component {
 
   signOut = async () => {
     try {
-      await AsyncStorage.removeItem('token');
-      Actions.login();
+      UserUtil.signOut((result, error) => {
+        AsyncStorage.removeItem('token', () => { Actions.login(); });
```

Comment:
ok. you put actions.login as callback upon completion of removeItem. However, you still better to put await because there might be a race condition.